### ISSUE TITLE
Storybook not gracefully exiting

### ISSUE THREAD

**Issue by sunlanterns** [storybooks/storybook#5393]
**Describe the bug**
When running storybook from the command line, trying to close the process using `ctl+C` or `ctrl+Z` it doesn't properly close the process so that the next time I try to run it, the port is still in use and I have to manually kill the storybook process.

**To Reproduce**
1. Run start-storybook with the cli
2. Stop the process using `ctl+C` or `ctrl+Z`
3. Try running start-storybook again

**Expected behavior**
Expected storybook to properly kill its own process so I'd be able to easily restart storybook.

**System:**
 - OS: MacOS
 - Device: MacBook
 - Browser: Chrome
 - Framework: Angular
 - Version: 4.1.6

**Comment by Keraito** [comment]
Hey @sunlanterns, that doesn't sound right. I couldn't quite reproduce the error in the Storybook repo, do you have a reproduction or a video that you can share to display the error? Also, through what command are you specifically starting storybook?

**Comment by anru** [comment]
I have same issue
![image](https://user-images.githubusercontent.com/130013/53876918-85cd7600-4010-11e9-9f33-03fd09508e3d.png)

**Comment by anru** [comment]
Solved issue by upgrading babel from 6 to 7 version

**Comment by harobed** [comment]
Same issue with this dependencies:

```
  "devDependencies": {
    "@babel/core": "7.5.4",
    "babel-loader": "8.0.6",
    "webpack": "4.39.2",
    "react": "16.9.0",
    "react-dom": "16.9.0",
    "react-router": "5.0.1",
    "react-router-dom": "5.0.1",
    "react-helmet": "5.2.1",
    "prop-types": "15.7.2",
    "react-dropdown": "1.6.4",
    "react-graceful-image": "1.2.9",
    "react-i18next": "10.12.2",
    "react-placeholder": "3.0.2",
    "reset-css": "4.0.1",
    "gatsby": "2.13.67",
    "gatsby-image": "2.2.8",
    "gatsby-plugin-manifest": "2.2.5",
    "gatsby-plugin-offline": "2.2.6",
    "gatsby-plugin-react-helmet": "3.1.3",
    "gatsby-plugin-sharp": "2.2.12",
    "gatsby-source-filesystem": "2.1.9",
    "gatsby-transformer-sharp": "2.2.6",
    "gatsby-plugin-client-side-redirect": "0.0.2",
    "@emotion/core": "10.0.16",
    "@emotion/styled": "10.0.15",
    "emotion": "10.0.14",
    "@svgr/cli": "4.3.2",
    "@fortawesome/fontawesome-svg-core": "1.2.21",
    "@fortawesome/free-brands-svg-icons": "5.10.1",
    "@fortawesome/free-solid-svg-icons": "5.10.1",
    "@fortawesome/react-fontawesome": "0.1.4",
    "i18next": "17.0.11",
    "i18next-browser-languagedetector": "3.0.3",
    "i18next-scanner": "2.10.2",
    "@storybook/addons": "5.1.9",
    "@storybook/addon-actions": "5.1.9",
    "@storybook/addon-links": "5.1.11",
    "@storybook/react": "5.1.9",
    "@storybook/addon-storysource": "5.1.10",
    "@storybook/addon-viewport": "5.1.10",
    "storybook-addon-i18next": "1.2.0",
    "storybook-react-router": "1.0.5",
    "eslint": "6.1.0",
    "eslint-plugin-react": "7.14.3"
  }
```

**Comment by harobed** [comment]
Bug fix after node upgrading from `12.9.0` to `12.9.1`.
